### PR TITLE
Opal 409

### PR DESCRIPTION
**Pull request by BE-USAF**

### CODE REVIEW

**Review by hh-usaf — APPROVED** [review]
Code looks neat and the pytests run correctly.

File: weave/index/index_sqlite.py [hunk 1]
```diff
@@ -99,60 +100,11 @@ def generate_index(self, **kwargs):
...
-        index_columns = ", ".join(index_columns)
-
-        bad_baskets = []
         for basket_json_address in basket_jsons:
-            with self.file_system.open(basket_json_address, "rb") as file:
-                basket_dict = json.load(file)
-
-                # If the basket is invalid, add it to a list, then skip it.
-                if not validate_basket_dict(basket_dict):
-                    bad_baskets.append(os.path.dirname(basket_json_address))
```
Comment (BE-USAF) [inline weave/index/index_sqlite.py:115]: I didn't know what was wrong here, so I just mirrored how we're doing it in the SQL index and offload most of this logic to the create_index_from_fs function

**Review by CW-USAF — APPROVED** [review]
Looks good and works on Windows!

**Review by JA-USAF — COMMENTED** [review]
😎 🐐

**Review by JA-USAF — APPROVED** [review]

**Pull request merged**